store/postgres/user.go

```diff
@@ -18,37 +20,194 @@ import (
 )
 
 type User struct {
-	Id        string    `db:"id"`
-	Name      string    `db:"name"`
-	Email     string    `db:"email"`
-	Metadata  []byte    `db:"metadata"`
-	CreatedAt time.Time `db:"created_at"`
-	UpdatedAt time.Time `db:"updated_at"`
+	Id        string       `db:"id"`
+	Name      string       `db:"name"`
+	Email     string       `db:"email"`
+	Metadata  []byte       `db:"metadata"`
+	CreatedAt time.Time    `db:"created_at"`
+	UpdatedAt time.Time    `db:"updated_at"`
+	DeletedAt sql.NullTime `db:"deleted_at"`
 }
 
-const (
-	getUserQuery             = `SELECT id, name,  email, metadata, created_at, updated_at from users where id=$1;`
-	getUsersByIdsQuery       = `SELECT id, name,  email, metadata, created_at, updated_at from users where id IN (?);`
-	getCurrentUserQuery      = `SELECT id, name, email, metadata, created_at, updated_at from users where email=$1;`
-	createUserQuery          = `INSERT INTO users(name, email, metadata) values($1, $2, $3) RETURNING id, name, email, metadata, created_at, updated_at;`
-	listUsersQuery           = `SELECT id, name, email, metadata, created_at, updated_at from users;`
-	selectUserForUpdateQuery = `SELECT id, name, email, metadata, updated_at from users where id=$1;`
-	updateUserQuery          = `UPDATE users set name = $2, email = $3, metadata = $4, updated_at = now() where id = $1 RETURNING id, name, email, metadata, created_at, updated_at;`
-	updateCurrentUserQuery   = `UPDATE users set name = $2, metadata = $3, updated_at = now() where email = $1 RETURNING id, name, email, metadata, created_at, updated_at;`
-)
+func PageTokenizer(label string, uuid string) string {
+	if uuid == "" {
+		uuid = "00000000-0000-0000-0000-000000000000"
```

Comment:
i can see `00000000-0000-0000-0000-000000000000 ` was used before as well
how about making this as a constant ?